server/cmd_auth.go

```diff
@@ -110,6 +114,7 @@ func (cmd *Subscribe) Handle(conn Conn) error {
 	if err != nil {
 		return err
 	}
+	defer mbox.Close()
 
 	return mbox.SetSubscribed(true)
```

Comment:
Maybe this should be a `User` method instead, so that a mailbox is only opened on SELECT?

Reply:
I agree. I think I even mentioned that somewhere but this is out-of-scope of this PR.